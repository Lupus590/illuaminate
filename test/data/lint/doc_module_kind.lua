-- config: (lint (only doc:unknown-module-kind)) (doc (module-kinds (kind1 "Kind 1")))

--- @module[kind=kind1]

--- @module[kind=kind2]
