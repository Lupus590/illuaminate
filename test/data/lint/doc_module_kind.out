doc_module_kind.lua:[5:5-5:11]: Unknown module kind "kind2" [doc:unknown-module-kind]
   │
 5 │ --- @module[kind=kind2]
   │     ^^^^^^^
1 errors and 0 warnings
